src/main/webapp/pages/details.html

```diff
@@ -1,36 +1,68 @@
-<div id="mainDiv" ng-controller="detailsCtrl as detailsCtrl">
+<div id="mainDiv">
```

Comment:
It seems like there are not much general code for showing details and creating new idea. May be it makes sense to split this view into 2?